### ISSUE TITLE
New Features

### ISSUE THREAD

**Issue by fera765** [Sharcoux/rn-css#1]
Is it possible to add @Media Query, Animations, Transitions?

**Comment by Sharcoux** [comment]
Yes, media queries are planned. I just need to finish some stuff before I can finish this, but that should not be to hard.
For transitions and animations, well... That's probably doable, but probably not that easy. I'll see what I can do about it. The Animated API of React-Native is really not easy to use.

**Comment by fera765** [comment]
of an exhibition at cssta, he managed to work with animations and a lot of things ..

https://www.npmjs.com/package/cssta

**Comment by Sharcoux** [comment]
Interesting.
It will definitely help.
As I said, I'm convinced it is doable. It is just a bit challenging because there are a lot of cases to handle. But the current way rn-css is handling units and props should make it not too hard to integrate animations and transitions.

**Comment by fera765** [comment]
I really liked the result of rn-css, I used it today to test and I really liked ...

It seems to be an incredible proposal

**Comment by Sharcoux** [comment]
Thank you =)
Don't hesitate if you encounter issues. The project is still very young but we are using it on a huge app so I can ensure that it will be maintained.
